sdk-tests/src/test/java/io/dapr/it/methodinvoke/grpc/MethodInvokeIT.java

```diff
@@ -138,10 +136,12 @@ public void testInvokeException() throws Exception {
             DaprException exception = assertThrows(DaprException.class, () ->
                 client.invokeMethod(daprRun.getAppName(), "sleep", req.toByteArray(), HttpExtension.POST).block());
 
-            assertEquals("UNKNOWN", exception.getErrorCode());
             if (this.useGrpc) {
-                assertEquals("UNKNOWN: ", exception.getMessage());
+                assertEquals("INTERNAL", exception.getErrorCode());
```

Comment:
I am looking into this too.

Reply:
Resolve the conflicts from master, please. For that, you can just pick the master changes.